### ISSUE TITLE
Failing to collapse sliproad

### ISSUE THREAD

**Issue by willwhite** [Project-OSRM/osrm-backend#4577]
Found another case where OSRM is failing to collapse a slip road here:

https://www.mapbox.com/get-directions/#17.36/37.29941/-121.83948?coordinates=-121.83803529740746,37.299429295255706;-121.83948370888538,37.29940937280185

**Comment by miccolis** [comment]
@oxidase or @MoKob can you tell it this related to scenario 1 (the only remaining open one) on https://github.com/Project-OSRM/osrm-backend/issues/4348 ?

**Comment by oxidase** [comment]
@miccolis atm the issue is not fixed in master. The problem is that sliproad obviousness check fails at https://github.com/Project-OSRM/osrm-backend/blob/d23a5fcfc4b2cb6cd899d2e4e09ff31d6416ccf6/include/extractor/guidance/intersection_handler.hpp#L508-L511 with 
```
(gdb) p intersection[i].angle
$17 = 158.95050803358333
(gdb) p turn_deviation
$18 = 21.049491966416667
(gdb) p FUZZY_ANGLE_DIFFERENCE
$19 = 25
```

It is fixed in fix/obvious-2 branch of #4426 
![screenshot from 2017-10-06 17-17-01](https://user-images.githubusercontent.com/4421046/31284942-4fc1c9b8-aaba-11e7-9179-b5d7b2358896.png)

**Comment by oxidase** [comment]
Fixed in 5.17.0
![screenshot from 2018-04-25 15-30-10](https://user-images.githubusercontent.com/4421046/39248881-962935ae-489d-11e8-9e63-957b81ddc0d5.png)
